symfony/amazon-mailer/4.3/manifest.json

```diff
@@ -1,6 +1,8 @@
 {
     "env": {
-        "#1": "MAILER_DSN=smtp://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses",
-        "#2": "MAILER_DSN=http://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses"
+        "#1": "AWS_ACCESS_KEY=",
+        "#2": "AWS_SECRET_KEY=",
+        "#3": "MAILER_DSN=smtp://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses",
+        "#4": "MAILER_DSN=http://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses"
```

Comment:
do we have any reco for smtp vs http?
looks like provide a choice is calling for trouble :)

Reply:
It was discussed a bit here: #594 - though I didn't see any docs that would recommend using HTTP if available, otherwise SMTP. Probably add a comment above these env vars explaining it?

Reply:
I would recommend using SMTP for now. That way, you will have the "same" experience across the board (from a local SMTP server on your dev machine to production via a provider).